core/src/ru/spbau/blackout/entities/DynamicObject.java

```diff
@@ -80,15 +86,34 @@ public void updateGraphics(float delta) {
     }
 
     @Override
-    public void updateForFirstStep() {
+    public void updateBeforeFirstStep() {
+        super.updateBeforeFirstStep();
+        // apply friction
+        if (!Utils.isZeroVec(velocity)){
```

Comment:
As in the previous case. It doesn't seem logical for me. `DynamicObject` still may do something here besides applying friction, even if in the current version it doesn't.